http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20211116_update_persistent_disk_formattedBy_enum.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="sshah" id="update_persistent_disk_formattedBy_enum">
+        <modifyDataType tableName="PERSISTENT_DISK" columnName="formattedBy" newDataType="ENUM('GALAXY', 'GCE', 'CROMWELL', 'CUSTOM')"/>
```

Comment:
Question for IA: So does this mean that all CUSTOM apps have to use the same persistent disk instance?

Reply:
The reason I added `CUSTOM` to this enum was so that custom apps can be marked `CUSTOM` in DB. Up until know I think it was default to `GALAXY`. If that was on purpose, I can remove this change.